Tessian/Tessian.manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "author": "Tessian",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "View metrics and charts based on Tessian data feeds.",
+    "helpURL": "https://tessian.zendesk.com/hc/en-us/articles/360004819057-What-is-the-Tessian-API-",
```

Comment:
Yes, but unfortunately is only visible for users with a Tessian account. I can check if we have a public version of the article.

Reply:
gotcha or you can give the url of the github Readme as well. Some partners do that.

Reply:
that will work, thanks!

Reply:
Thanks @fede-tessian, yes we require docs on how to install/troubleshoot/etc be made public so our field teams can help joint customers w/o having to reach out and get Tessian accounts. Most partners just update the README.md file with this info here in the repo since it's public. Thanks!